### PR TITLE
box with legacy info in footer

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**

### CODE REVIEW

File: docs/_templates/footer.html [hunk 1]
```diff
@@ -17,6 +17,16 @@
 
     <hr/>
 
+    <div class="legacy-box">
+        <h1 style="color:#4e566d; text-align: center;">WARNING: This is documentation for Legacy Neptune client library</h1>
+        <p style="color:#4e566d; text-align: center;">This is the documentation of the legacy client library which is under the maintenance support only.</p>
+        <p style="color:#4e566d; text-align: center;">No new updates will be made to this documentation and legacy client library.</p>
+        <p style="color:#4e566d; text-align: center;">It is highly recommended going to the <a href="https://docs.neptune.ai"> new documentation</a>.</p>
+        <h2 style="color:#4e566d; text-align: center;"><b>You can follow this <a href="https://docs.neptune.ai/links/migration-guide"> migration guide</a> to update your code to the version of the Neptune client library.</b></h2>
```
Comment (Herudaio) [inline docs/_templates/footer.html:25]: > the version
-> the new versios?

**Review by Herudaio — CHANGES_REQUESTED** [review]
Single change request, apart from that looks good. After change can merge :)

File: docs/_templates/footer.html (hunk 1 above)
Comment (kamil-kaczmarek) [inline docs/_templates/footer.html:25]:
```suggestion
        <h2 style="color:#4e566d; text-align: center;"><b>You can follow this <a href="https://docs.neptune.ai/links/migration-guide"> migration guide</a> to update your code to the new version of the Neptune client library.</b></h2>
```

Comment (kamil-kaczmarek) [reply docs/_templates/footer.html]: fixed

**Pull request merged**